skills-src/delegation/references/implementer-prompt.md

```
@@ -345,6 +357,18 @@ pwd | grep -q "\.worktrees" || { echo "ERROR: Not in worktree!"; exit 1; }
 
 This check prevents accidental modifications to the main project root, which would cause merge conflicts with other parallel tasks.
 
+## CRITICAL: Base Verification (MANDATORY)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)